ansible/roles/ingress/templates/ingress.yml.j2

```
@@ -27,6 +27,14 @@ metadata:
     nginx.ingress.kubernetes.io/server-snippet: |
       client_header_buffer_size 100k;
       large_client_header_buffers 4 100k;
+      proxy_intercept_errors on;
+{% if KEDA_NAMESPACE_ACTIVATOR_ENABLED is defined %}
+      error_page 503 = @errorpages;
+
+      location @errorpages {
+        return 302 'https://activate.cd.dbildungscloud.dev/namespace?namespace={{ NAMESPACE }}&redirected-from-503=true';
+      }
+{% endif %}
```

The proxy_intercept_errors shoud only add if the namespace activator is there and not at gerneral level 
```suggestion
{% if KEDA_NAMESPACE_ACTIVATOR_ENABLED is defined %}
      proxy_intercept_errors on;
      error_page 503 = @errorpages;

      location @errorpages {
        return 302 'https://activate.cd.dbildungscloud.dev/namespace?namespace={{ NAMESPACE }}&redirected-from-503=true';
      }
{% endif %}
```